optimade/server/query_params.py

```diff
@@ -214,7 +213,7 @@ def __init__(
             ge=0,
         ),
         page_number: int = Query(
-            0,
+            None,
```

Comment:
Do we need to change the other `page_<param>` fields to default to `None` too? If you don't have strong feelings, I would suggest keeping this at 0 so that the schema does not change (it seems too inconsequential to bump the version for)

Reply:
Somehow I had not realized that 0 would evaluate to False too, so we can indeed keep the default as 0.